app/src/test/java/ch/hikemate/app/model/route/HikeRoutesRepositoryOverpassTest.kt

```diff
@@ -474,7 +589,7 @@ class HikeRoutesRepositoryOverpassTest {
     }
 
     hikingRouteProviderRepositoryOverpass.getRoutes(
-        bounds, { routes -> assertEquals(simpleRoutes, routes) }) {
+        bounds, { routes -> assertEquals(alternativeRoutes, routes) }) {
```

Comment:
Question: Why did you changed this ?

Reply:
Looks like I was tired and duplicated some code